theme/index.tsx

```diff
@@ -164,18 +165,43 @@ function HomeLayout() {
     return () => clearInterval(ticker);
   }, [updateText, delta, page]);
 
+  const { pre: PreWithCodeButtonGroup, code: Code } = getCustomMDXComponent();
+
+  // Rspress would pass `afterHero: undefined` and `afterHeroActions: undefined` props to HomeLayout,
+  const {
+    afterHero = (
+      <>
+        <Features src={routePath} /> {routePath === '/' && <ShowCase />}
+        {routePath === '/' && <Banner />}
+      </>
+    ),
+    afterHeroActions = (
```

Comment:
<img width="339" height="83" alt="Image" src="https://github.com/user-attachments/assets/895d42d4-d19e-481e-90a7-41440d308fc7" />

Add 50% transparency to make this block blend better with the background.

Reply:
<img width="444" height="333" alt="image" src="https://github.com/user-attachments/assets/30bbe252-611d-4d43-bfef-c5f976b5aaad" />